scripts/inner/kubestellar-kube-bind

```diff
@@ -61,36 +75,55 @@ then
     exit 1
 fi
 
-space_conf_file="/tmp/$space_name.conf"
-kubectl-kubestellar-get-config-for-space --space-name $space_name --sm-core-config $SM_CONFIG --sm-context $SM_CONTEXT $in_cluster --output $space_conf_file
-KUBECONFIG=$space_conf_file
-
-if kubectl get crd $resource_to_bind.edge.kubestellar.io &> /dev/null; then
-    echo "CRD for $resource_to_bind already in place"
-    exit 0
-fi
+set -e
 
-mkdir -p $kb_dir
-konnector_log_file=$kb_dir/kube-bind-konnector-$space_name.log
+konnector_dir="konnector-${space_name}"
+konnector_log_file="${konnector_dir}/log"
+finally=""
 
 if pgrep konnector | xargs ps e -p 2>/dev/null | egrep "KB_CONSUMER=$space_name(\s|$)" &> /dev/null; then
     echo "kube-bind konnector for space $space_name already running"
+    consumer_kubeconfig=$(mktemp "$space_name".kubeconfig#XXXX)
+    finally="rm -f '$consumer_kubeconfig'; "
+    trap "$finally" EXIT
+    launch_konnector=false
 else
-    if ! kubectl get namespace kube-system &> /dev/null
-    then kubectl create namespace kube-system &> /dev/null
+    mkdir -p "$konnector_dir"
+    consumer_kubeconfig="${konnector_dir}/kubeconfig"
+    launch_konnector=true
+fi
+
+kubectl kubestellar space get-kubeconfig "${sub_flags[@]}" "${kubectl_flags[@]}" $space_name "$consumer_kubeconfig"
```

Comment:
I see you replaced explicit flag based arguments with positional arguments.  I find positional arguments very confusing and hard to maintain

Reply:
When there are a lot of them, yes. When there are only two, I think it is quite manageable.